config/foreman-answers.yaml

```diff
@@ -12,6 +12,7 @@
 foreman:
   custom_repo: true
   email_config_method: database
+  dynflow_in_core: true
```

Comment:
The whole dynflow_executor change won't end up in any component used by 1.14 releases, or am I getting something completely wrong?